.github/workflows/integration-tests.yml

```diff
@@ -58,8 +58,7 @@ jobs:
           file: src/Dockerfile
           platforms: linux/${{ matrix.arch }}
           tags: |
-            ${{ env.VERSIONED_IMAGE_FQN }}
-            ${{ env.LATEST_IMAGE_FQN }}
+            netwatch_ssh-attackpod:latest-${{ matrix.arch }}
```

Comment:
please don't. 
would prefer to keep both: a versioned one, and the latest alias.

image name is derived from the repo name (so builds on forks would work as well), no need to hard-code it.

Reply:
Hi @heubeck,
Thank you for your feedback. I think we can remove the complete ``Build image repo name`` section. As it is the workflow just for the tests not the publishing the containers to a registry.

Reply:
ah - you're right. pardon me, missed which workflow this is ;)